tests/Doctrine/Tests/DBAL/Platforms/OraclePlatformTest.php

```diff
@@ -356,6 +362,8 @@ public function testGenerateTableWithAutoincrement() : void
 
     /**
      * {@inheritDoc}
+     *
+     * @psalm-return array
```

Comment:
Does Psalm complain about the return type given in the parent?

Reply:
It does complain in my IDE, maybe that's a complain at a higher level? Anyway, I think I am going to remove this and use `never-return|string[]` in the parent, see https://github.com/phpstan/phpstan/issues/3335 . WDYT?

Reply:
In fact, the only version I could get to work on both most recent versions of both tools is that one:
- https://psalm.dev/r/01f47ec513
- https://phpstan.org/r/87a19063-3ef0-4a8a-9c1c-301d662d0090

Reply:
Maybe I should report this: https://psalm.dev/r/01f47ec513

Reply:
I asked for help at https://symfony-devs.slack.com/archives/C8SFXTD2M/p1590518449028600

Reply:
Ok, here is a commit that should help with Psalm https://github.com/vimeo/psalm/commit/f0a5bd74b6f81c40393d45cd3490a5d2721509c2 :)

Reply:
I updated Psalm, and I can drop the commit about never return :tada: 

But now there are new issues to fix :see_no_evil:

Reply:
It was expected after an update :smile: 

Nice work.